src/app/components/modules/simplified-lunch/simplified-lunch.component.html

```diff
@@ -0,0 +1,55 @@
+<div class="module-container">
+<div *ngIf="loading" class="loading-container">
+	<div mymicds-blur class="loading">
+		<h1>Fetching the Lunch Menu...</h1>
+	</div>
+</div>
+<div *ngIf="!loading" class="lunch-container container">
+
+
+	<!-- Actual Lunch --> 
+	<div class="lunch-week" routerLink='/lunch'>
+		<!-- Loop through Weekdays -->
+		<div myMicds-blur class = "lunch-day text-center" [class.today] = "">
+		<!-- <div mymicds-blur *ngFor="let day of lunch" class="lunch-day" [class.today]="day.date.today">
```

Comment:
Try to take out any commented out code before merging, unless you are truly wanting to use it at a later time. Since this is copied from the original lunch page, I'd say it's even less of a concern to have it commented here because you can copy it over again.